server/bin/pbench-server-prep-shim-002.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import os
+import sys
+import glob
+import re
+import errno
+import hashlib
+import shutil
+import subprocess
+import tempfile
+import tarfile
+import pathlib
+from enum import Enum
+from argparse import ArgumentParser
+from s3backup import S3Config, Entry
+
+from pbench import PbenchConfig, BadConfig, get_pbench_logger, quarantine, rename_tb_link, md5sum
+from pbench.report import Report
+
+_NAME_ = "pbench-server-prep-shim-002"
+
+def main():
+    cfg_name = os.environ.get("_PBENCH_SERVER_CONFIG")
+    if not cfg_name:
+        print("{}: ERROR: No config file specified; set CONFIG".format(_NAME_))
+        return 2
+
+    try:
+        config = PbenchConfig(cfg_name)
+    except BadConfig as e:
+        print("{}: {} (config file {})".format(_NAME_, e, cfg_name), file=sys.stderr)
+        return 1
+
+    if not os.path.exists(config.ARCHIVE):
+        print("{}: Bad ARCHIVE={}".format(_NAME_, config.ARCHIVE), file=sys.stderr)
+        return 1
+
+    logger = get_pbench_logger(_NAME_, config)
+
+    qdir = config.QDIR = config.get('pbench-server', 'pbench-quarantine-dir')
+    if len(qdir) == 0:
+        logger.error("Failed: getconf.py pbench-quarantine-dir pbench-server")
+        return 2
+
+    if not os.path.isdir(qdir):
+        logger.error("Failed: {} does not exist, or is not a directory".format(qdir))
+        return 2
+
+    receive_dir_prefix = config.get('pbench-server','pbench-receive-dir-prefix')
+    if len(receive_dir_prefix)==0:
+        logger.error("Failed: getconf.py pbench-receive-dir-prefix pbench-server")
+        return 2
+
+    receive_dir= receive_dir_prefix+"-002"
```

Comment:
It is a good practice not to use `+` for sting concatenation, use f-string instead:
f"{receive_dir_prefix}-002"

Reply:
ok